packages/vue-app/template/index.js

```diff
@@ -92,10 +92,10 @@ function registerModule (path, rawModule, options = {}) {
 <% } %>
 
 async function createApp(ssrContext, config = {}) {
-  const router = await createRouter(ssrContext, config)
+  const store = <%= store ? %> createStore(ssrContext) <% : undefined %>
+  const router = await createRouter(ssrContext, config, store)
```

Comment:
Just wondering here. We did not pass the context to the `createRouter` function, is it a different context expected than the one provided to the createApp method?

Reply:
> Is it a different context expected than the one provided to the createApp method?

Yes. ssrContext is different than nuxt context (and also slightly from app one).

I think would be safer to pass store as an object to avoid adding 4th param if we had any similar requirement in the future:

```suggestion
  const router = await createRouter(ssrContext, config, { store })
```